modules/setup-bias-monitoring.adoc

```diff
@@ -0,0 +1,83 @@
+:_module-type: PROCEDURE
+
+[id="setup-bias-monitoring_{context}"]
+= Before you begin
+
+== Downloading tutorial files
+. Go to https://github.com/trustyai-explainability/odh-trustyai-demos/tree/main.
+. Click the *Code* button and then click *Download ZIP* to download the repository.
+. Extract the downloaded repository files.
+
+== Enabling user-workload-monitoring
+To get enable Open Data Hub’s monitoring stack, you must enable monitoring for user-defined projects.
+
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI:
```

Comment:
suggest you move and rewrite lines 14-22 to a new section - such as:
**Logging in to the OpenShift cluster from the command line**

1. Obtain the command for logging in to the OpenShift cluster from the command line:
 a. In the upper-right corner of the OpenShift web console, click your user name and then select **Copy login command**.
b. Login with your credentials and then click **Display token**.
c. After you log in with your credentials, click **Display token**.
d. Copy the Log in with this token command which has the following syntax:
  oc login --token=<token> --server=<openshift_cluster_url>
2. Open a terminal window and paste the login command.